src/rpc/methods/eth.rs

```diff
@@ -320,14 +318,15 @@ pub async fn eth_get_balance<DB: Blockstore>(
     Ok(BigInt(actor.balance.atto().clone()))
 }
 
-pub async fn eth_syncing<DB: Blockstore>(
+pub async fn eth_syncing<DB: Blockstore + Sync + Send + 'static>(
     _params: Params<'_>,
     data: Ctx<DB>,
 ) -> Result<LotusJson<EthSyncingResult>, ServerError> {
-    let RPCSyncState { active_syncs } = sync_state(data).await?;
+    let crate::rpc::sync::RPCSyncState { active_syncs } =
+        crate::rpc::sync::SyncState::handle(data.clone(), ()).await?;
     match active_syncs
-        .iter()
-        .rev()
```

Comment:
Do we not need `.rev()` anymore?

Reply:
I think that would change the logic, no?

Reply:
good catch!